src/lib/deploy-helpers.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { K8s, kind, KubernetesListObject } from "kubernetes-fluent-client";
+import Log from "./logger";
+
+// get all deployments
+export async function getDeployments() {
+  const deployments = await K8s(kind.Deployment).InNamespace("pepr-system").Get();
+  return deployments;
+}
+
+// waits for all replicas to be ready for all deployments
+export async function waitForAllDeploymentReplicas(deployments: KubernetesListObject<kind.Deployment>) {
+  let status = false;
+  let readyCount = 0;
+
+  if (deployments.items.length === 0) {
```

Comment:
```suggestion
  if (deployments.items.length === 0) {
```
So if there are no deployments then it returns false and says `✅ Module deployed successfully`? I think we need to revisit this. This may be an error but definitely not false

Reply:
im not sure we need to have check the more that I think about it

Reply:
Removed deployments length check.